integration/kafka/src/main/java/alluxio/kafka/connect/format/AlluxioFormat.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.kafka.connect.format;
+
+import alluxio.client.file.FileOutStream;
+
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import java.io.IOException;
+
+/**
+ * AlluxioFormat Interface describes the data format which stored in Alluxio.
+ */
+public interface AlluxioFormat {
+
+  /**
+   * Get File extension name.
+   *
+   * @return File extension name
+   */
+  public String getExtension();
```

Comment:
NIT: inside an interface definition, we typically don't have `public` as they are redundant modifiers on interfaces